system/timezone.py

```diff
@@ -110,7 +111,7 @@ def __init__(self, module):
         # Initially there's only info of "planned" phase, but the
         # `self.check()` function will fill out it.
         self.value = dict()
-        for key in module.argument_spec.iterkeys():
+        for key in iterkeys(module.argument_spec):
```

Comment:
Stylewise we don't need to do this.  Iterating over the dict is mostly the same as iterkeys.

``` python
for key in module.argument_spec:
```